### PR TITLE
[MFTF] Refactoring use ActionGroup  instead <see/> tag

### PR DESCRIPTION
**Pull request by Usik2203**
### Description (*)

This PR created and use `AssertStorefrontProductInfoMainProductNameActionGroup` instead  
 `<see selector="{{StorefrontProductInfoMainSection.productName}}" userInput="Woops...." stepKey="assertElement"/>`
 Regarding MFTF Best Practices

### CODE REVIEW

**Comment by Usik2203** [comment]
@magento run all tests

**Comment by Usik2203** [comment]
@magento run all tests

**Comment by Usik2203** [comment]
@magento run Functional Tests B2B  , Integration Tests  , Static Tests

**Review by Den4ik — CHANGES_REQUESTED** [review]
@Usik2203 Thanks for the update.
I found 69 files that contains `see StorefrontProductInfoMainSection.productName` usage.
Could you please update all files?

File: app/code/Magento/Catalog/Test/Mftf/Test/AdminDeleteConfigurableChildProductsTest.xml [hunk 1]
```diff
@@ -129,6 +134,11 @@
             <argument name="productName" value="$$createConfigProduct.name$$"/>
         </actionGroup>
         <!--Open Product Store Front Page and Verify Product is Out Of Stock -->
+        <amOnPage url="$$createConfigProduct.custom_attributes[url_key]$$.html" stepKey="openProductInStoreFront1"/>
+        <waitForPageLoad stepKey="waitForProductToLoad1"/>
+        <seeElement selector="{{StorefrontHeaderSection.NavigationCategoryByName($$createCategory.name$$)}}" stepKey="seeCategoryInFrontPage1"/>
+        <actionGroup ref="AssertStorefrontProductInfoMainProductNameActionGroup" stepKey="seeProductNameInStoreFront1">
+            <argument name="value" value="$$createConfigProduct.name$$"/>
```
Comment (eduard13) [inline app/code/Magento/Catalog/Test/Mftf/Test/AdminDeleteConfigurableChildProductsTest.xml]: Hi @engcom-Charlie, looks like something went wrong on the last commit while merging the mainline. These changes shouldn't be here.

**Review by eduard13 — CHANGES_REQUESTED** [review]

File: app/code/Magento/Catalog/Test/Mftf/Test/AdminDeleteConfigurableChildProductsTest.xml (hunk 1 above)
Comment (engcom-Charlie) [reply app/code/Magento/Catalog/Test/Mftf/Test/AdminDeleteConfigurableChildProductsTest.xml]: HI @eduard13, thanks for updating. The merged issue has been fix now.

**Comment by engcom-Charlie** [comment]
Hi @Den4ik,

I have replaced the "see StorefrontProductInfoMainSection.productName"  with AssertStorefrontProductInfoMainProductNameActionGroup action group in the all the remaining files where ever its required.

Can you please review the PR now.

Thank you!

**Comment by Den4ik** [comment]
@magento run all tests

**Review by sidolov — CHANGES_REQUESTED** [review]
@Usik2203 please, check failed tests

**Comment by engcom-Charlie** [comment]
@magento run all tests

**Comment by eduard13** [comment]
@magento run Functional Tests EE

**Comment by engcom-Charlie** [comment]
Hi @sidolov,

I have updated this PR branch with latest 2.4 develop code. Now, only one test is failing and I have checked that the test case which is failing here is not part of this PR changes. Or not failing because of any changes made in this PR.

Hence, moving this PR in review.

Thank you!

**Review by sidolov — CHANGES_REQUESTED** [review]
@Usik2203  please, resolve merge conflicts

**Comment by engcom-Charlie** [comment]
Hi @Usik2203 

Thanks for the contribution!

I will look into resolving conflicts.

Thank you!

**Comment by engcom-Charlie** [comment]
@magento run all tests

**Comment by engcom-Charlie** [comment]
@magento run all tests

**Comment by engcom-Charlie** [comment]
Hi @sidolov,

I have merged the latest 2.4-develop into this PR branch and resolevd all conflicts.

The below test cases which are failing in build are not part of this PR changes and also not failing because of any changes made in this PR. Hence moving this PR to Merge in Progress.

Failing tests are:
Functional Test EE
app/code/Magento/CurrencySymbol/Test/Mftf/Test/AdminCheckCurrencyConverterApiConfigurationTest.xml
app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreditMemoAutomaticallyReturnCreditMemoItemToStockForWholeOrderWithSimpleProductOnDefaultStockInAdminTest.xml

Functional CE:
app/code/Magento/CurrencySymbol/Test/Mftf/Test/AdminCheckCurrencyConverterApiConfigurationTest.xml

Functional B2B:
app/code/Magento/CurrencySymbol/Test/Mftf/Test/AdminCheckCurrencyConverterApiConfigurationTest.xml
app/code/Magento/ConfigurableProduct/Test/Mftf/Test/StorefrontGalleryConfigurableProductWithVisualSwatchAttributePrependMediaTest.xml


Thank you!

**Pull request merged**